src/main/java/rs117/hd/environments/EnvironmentManager.java

```diff
@@ -219,6 +219,20 @@ public void update()
 		lastFrameTime = System.currentTimeMillis();
 		lastSkyColor = config.defaultSkyColor();
 		lastEnvironmentLighting = config.atmosphericLighting();
+
+		// If menu and environment overrides are active, set skybox to selected color. Otherwise, set to environment color
+		if(config.overrideSky() && currentEnvironment.isAllowSkyOverride())
+		{
+			DefaultSkyColor defaultSkyColor = config.defaultSkyColor();
+			if (defaultSkyColor != DefaultSkyColor.DEFAULT)
+			{
+				targetFogColor = new float[]{defaultSkyColor.getR() / 255f, defaultSkyColor.getG() / 255f, defaultSkyColor.getB() / 255f};
+			}
```

Comment:
`lastSkyColor` (defined a few lines up) seems to be the same as your `defaultSkyColor`

```suggestion
			if (lastSkyColor != DefaultSkyColor.DEFAULT)
			{
				targetFogColor = new float[]{lastSkyColor.getR() / 255f, lastSkyColor.getG() / 255f, lastSkyColor.getB() / 255f};
			}
```

Reply:
Fixed thx